specification/common-types/resource-management/v6/privatelinks.json

```diff
@@ -121,7 +161,7 @@
       },
       "allOf": [
         {
-          "$ref": "../v5/types.json#/definitions/Resource"
+          "$ref": "./types.json#/definitions/Resource"
```

Comment:
I assume this is an intentional change to which file is referenced, but just checking.

Reply:
yes, it is intentional as the file is on the same folder and on every version the file is going to be generated